### PR TITLE
PowerPC: Minor cleanups.

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

**Comment by FioraAeterna** [comment]
wow, talk about ugly code. this looks good.

**Comment by lioncash** [comment]
Because it got removed in the rebase:

> Tilka added a note 10 minutes ago 
> Don't we have an enum like this already? I'm not sure if it was merged...

@Tilka Yeah, there was a previous enum like that. I updated the commit to get rid of it.

File: Source/Core/Core/PowerPC/PPCTables.cpp [hunk 1]
```diff
@@ -93,68 +94,23 @@ Interpreter::_interpreterInstruction GetInterpreterOp(UGeckoInstruction _inst)
...
-		case 759:
-		case 983:
-			return true;
-		default:
-			return false;
-		}
-	default:
+	GekkoOPInfo* const info = GetOpInfo(inst);
+
+	if (info == nullptr)
```
Comment (magumagu) [inline Source/Core/Core/PowerPC/PPCTables.cpp]: info is never null.

Comment (lioncash) [inline Source/Core/Core/PowerPC/PPCTables.cpp]: Modified accordingly

**Pull request merged**